### PR TITLE
LRQA-71834 | master

### PR DESCRIPTION
**Pull request by melodywu**
https://issues.liferay.com/browse/LRQA-71834
I ran them here [melodywu#30](https://github.com/melodywu/liferay-portal/pull/30).

### CODE REVIEW

**Comment by melodywu** [comment]
ci:test:sf

File: modules/apps/account/account-test/src/testFunctional/tests/Accounts.testcase [hunk 1]
```diff
@@ -167,15 +167,15 @@ definition {
 
 		LexiconEntry.gotoEntry(rowEntry = "Person Account");
 
-		JSONAccountEntryUser.addAccountEntryUser(
-			accountEntryName = "Person Account",
+		Accounts.addNewUserToPersonAccount(
+			accountName = "Person Account",
```
Comment (patriciajeanperez) [inline modules/apps/account/account-test/src/testFunctional/tests/Accounts.testcase:171]: @melodywu moving this back to use UI should have been in the previous commit since these changes are not related to this commit. 

also, were we only going to add the user to the person account via UI the second time? did we have to use UI for both times?

**Review by patriciajeanperez — COMMENTED** [review]
@melodywu Please see minor comments, thanks! I will also run the Accounts tests for a few times to make sure they are still stable since we are also updating acceptance tests.

I also created a new test refactor ticket: https://issues.liferay.com/browse/LRQA-72320 that I discovered while reviewing this PR, but it can be a separate work from this ticket. Just fyi. Thanks!

**Comment by patriciajeanperez** [comment]
ci:test:account

**Comment by patriciajeanperez** [comment]
ci:test:account

**Comment by patriciajeanperez** [comment]
ci:test:account

File: modules/apps/account/account-test/src/testFunctional/tests/2BAccount.testcase [hunk 2]
```diff
@@ -1266,19 +1238,21 @@ definition {
...
 			userLastName = "user2",
 			userScreenName = "user2");
 
+		Navigator.gotoPage(pageName = "2B Account Page");
+
+		LexiconEntry.gotoEntryMenuItemNoError(
+			menuItem = "Manage Users",
+			rowEntry = "Account Name");
+
+		Accounts.gotoAssigneeTab(assigneeTab = "Users");
```
[the remark this answers is not in the document]
Comment (melodywu) [reply modules/apps/account/account-test/src/testFunctional/tests/2BAccount.testcase:1254]: Done Thanks！
